api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityAndEntitlementDecisionFactoryTest.java

```diff
@@ -0,0 +1,99 @@
+package uk.gov.dhsc.htbhf.claimant.service;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.dhsc.htbhf.claimant.entitlement.PaymentCycleVoucherEntitlement;
+import uk.gov.dhsc.htbhf.claimant.model.eligibility.EligibilityAndEntitlementDecision;
+import uk.gov.dhsc.htbhf.claimant.testsupport.ClaimantServiceIdentityAndEligibilityResponseTestDataFactory;
+import uk.gov.dhsc.htbhf.dwp.model.v2.IdentityAndEligibilityResponse;
+import uk.gov.dhsc.htbhf.eligibility.model.EligibilityStatus;
+
+import java.util.UUID;
+
+import static java.util.Collections.emptyList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.EligibilityAndEntitlementTestDataFactory.aValidDecisionBuilder;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.PaymentCycleVoucherEntitlementTestDataFactory.aPaymentCycleVoucherEntitlementWithVouchers;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.PaymentCycleVoucherEntitlementTestDataFactory.aPaymentCycleVoucherEntitlementWithZeroVouchers;
+import static uk.gov.dhsc.htbhf.dwp.testhelper.TestConstants.NO_HOUSEHOLD_IDENTIFIER_PROVIDED;
+import static uk.gov.dhsc.htbhf.dwp.testhelper.v2.IdentityAndEligibilityResponseTestDataFactory.anIdentityMatchedEligibilityNotConfirmedResponse;
+
+class EligibilityAndEntitlementDecisionFactoryTest {
+
+    private EligibilityAndEntitlementDecisionFactory factory = new EligibilityAndEntitlementDecisionFactory();
+    private IdentityAndEligibilityResponse eligibilityResponse =
+            ClaimantServiceIdentityAndEligibilityResponseTestDataFactory.anAllMatchedEligibilityConfirmedUCResponseWithHouseholdIdentifier();
```

Comment:
Cannot use a static import for this because the import statement is more than 160 lines and Checkstyle won't let you put a line break in an import statement. Sigh.

Reply:
Maybe `IdAndEligibilityResponseTestDataFactory`?

Reply:
Yes, that works, the static import line is now 159 chars, so perfect! :-)